### ISSUE TITLE
Support for the longranger align pipeline

### ISSUE THREAD

**Issue by olavurmortensen** [ewels/MultiQC#937]
I imagine this should be relatively straight forward because the longranger align pipeline is basically a subset of the longranger targeted/wgs pipeline. So should be a simple extension of #740.

**Tool Details**
 - Name of tool: longranger align
 - Tool homepage: https://support.10xgenomics.com/genome-exome/software/pipelines/latest/advanced/other-pipelines
 - Tool description: aligns linked-reads from 10x Genomics Chromium Controller. Basically a subset of the longranger targeted/wgs pipeline, which [MultiQC supports](https://multiqc.info/docs/#longranger).
 - Data suitable for MultiQC plot(s): Basically a subset of the MultiQC report for longranger targeted/wgs
 - Most interesting data for General Stats table: Alignment and duplicates.

**Tool output:**
 - Log filename pattern: summary.csv
 - File contents: I've uploaded an example in the link below. I've uploaded two files, one for the supported targeted/wgs pipeline, and one for the unsupported "align" pipeline. 

[tool_output.zip](https://github.com/ewels/MultiQC/files/3022684/tool_output.zip)

**Comment by ewels** [comment]
Hi @olavurmortensen,

Thanks for the request. As you say, the outputs look _very_ similar. In fact, when I changed the MultiQC search pattern to find both files it almost worked; I just had an error from trying to create a string (an empty value) to a float:

```python
Module longranger raised an exception: Traceback (most recent call last):
  File "/Users/ewels/GitHub/MultiQC/scripts/multiqc", line 440, in multiqc
    output = mod()
  File "/Users/ewels/GitHub/MultiQC/multiqc/modules/longranger/longranger.py", line 281, in __init__
    plot = table.plot(self.longranger_data, self.headers, config_table)
  File "/Users/ewels/GitHub/MultiQC/multiqc/plots/table.py", line 49, in plot
    return make_table ( dt )
  File "/Users/ewels/GitHub/MultiQC/multiqc/plots/table.py", line 138, in make_table
    val = header['modify'](val)
  File "/Users/ewels/GitHub/MultiQC/multiqc/modules/longranger/longranger.py", line 48, in <lambda>
    'modify': lambda x: float(x) * 100.0,
ValueError: could not convert string to float:
```

I just pushed a commit (d4a622d) to make the module more tolerant of missing values, and it now seems to work fine I think 😀 

Would you mind having a play with it to see if it works as you'd expect? Does the search pattern need updating? (See https://multiqc.info/docs/#longranger )

Cheers,

Phil

**Comment by ewels** [comment]
For convenience and comparison: [multiqc_report.html.zip](https://github.com/ewels/MultiQC/files/3025329/multiqc_report.html.zip)

**Comment by olavurmortensen** [comment]
Seems to work fine, I've uploaded the report (see below). It simply says `Error - was not able to plot data.` for the phasing plot, and it just shows no data for "Large SVs". This is fine with me, so as long as you're satisfied we can call this done :)
[multiqc_longranger_align.tar.gz](https://github.com/ewels/MultiQC/files/3025453/multiqc_longranger_align.tar.gz)

Thanks for addressing this so quickly!

**Comment by ewels** [comment]
Great! Thanks for testing. I've just pushed another commit that checks for the data that's present and skips the phasing report section if there's nothing to plot. I also updated the code so that it doesn't save any empty fields from the summary file. So now instead of an empty string (and a blank table cell), the columns are skipped properly if there's no data. This also means that if there's a mix of sample types as in the original test data, then the suffixes such as `%` and `Mb` are skipped properly.

Let me know how you get on 👍 

Phil
